docs/features.md

```diff
@@ -0,0 +1,54 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.Markdownlint addin.
+---
+The [Cake.Issues.Markdownlint addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from [Markdownlint] logfiles.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [MarkdownlintAddRuleUrlResolver] alias.
+
+# Supported log file formats
+
+* [MarkdownlintLogFileFormat] alias for reading issues from [Markdownlint] output generated with `options.resultVersion` set to 1.
+* [MarkdownlintCliLogFileFormat] alias for reading issues from [markdownlint-cli] log files.
+
+  :::{.alert .alert-info}
+  [markdownlint-cli] can be run with the [Cake.Markdownlint] addin.
+  :::
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.